Content.Shared/Administration/AdminFrozenSystem.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Content.Shared.Interaction.Events;
+using Content.Shared.Item;
+using Content.Shared.Movement;
+using Content.Shared.Throwing;
+
+namespace Content.Shared.Administration;
+
+public sealed class AdminFrozenSystem : EntitySystem
+{
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<AdminFrozenComponent, PickupAttemptEvent>((_, _, args) => args.Cancel());
+        SubscribeLocalEvent<AdminFrozenComponent, ThrowAttemptEvent>((_, _, args) => args.Cancel());
+        SubscribeLocalEvent<AdminFrozenComponent, AttackAttemptEvent>((_, _, args) => args.Cancel());
+        SubscribeLocalEvent<AdminFrozenComponent, MovementAttemptEvent>((_, _, args) => args.Cancel());
+        SubscribeLocalEvent<AdminFrozenComponent, InteractionAttemptEvent>((_, _, args) => args.Cancel());
```

Comment:
Apparently use-in-hand doesn't (yet) also check can interact. So `UseAttemptEvent` is needed to stop people from detonating grenades in their hands